src/pipeline_edsl/semantics.ml

```diff
@@ -173,6 +173,12 @@ module type Bioinformatics_base = sig
     [ `Bam ] repr ->
     [ `Bam ] repr
 
+  val picard_reorder_sam:
+    ?mem_param : string ->
```

Comment:
I was thinking about it, but for something which has only two params, I thought that would be heavyweight (and easy to introduce later if needed).